public/locales/en/translation.json

```diff
@@ -551,7 +553,8 @@
       "runnerNotUpdated": "Runner hasn't been updated",
       "twingraphNotCreated": "A problem occurred during twingraph creation",
       "twingraphStatusPollingFailed": "A problem occurred when querying the status of dataset {{datasetDetails}}",
-      "twingraphDataNotRestored": "Twingraph data hasn't been restored"
+      "twingraphDataNotRestored": "Twingraph data hasn't been restored",
+      "cannotAccessDatasetBases": "For the scenario \"{{scenarioName}} ({{scenarioId}})\", you don't have access to its bases dataset \"{{datasetBasesId}}\". Either the dataset no longer exists or you don't have sufficient permissions."
```

Comment:
Could we rephrase the label to this?
```
Some parts of the scenario \"{{scenarioName}}\" ({{scenarioId}}) could not be loaded, because the associated dataset \"{{datasetBasesId}}\" does not exist, or you don't have access to it.
```